api/albums.js

```diff
@@ -46,21 +49,43 @@ async function joinPendingAlbums(userToken, userId) {
   return Promise.all(joinAllPromises);
 }
 
-async function shareAlbum(userToken, userId, albumId) {
-  const endpoint = `albums/${albumId}:share`;
-  const params = { method: 'POST' };
-  const body = {
-    sharedAlbumOptions: {
-      isCollaborative: false,
-      isCommentable: true,
-    },
-  };
-  const result = await fetchJson({ body, endpoint, params, userToken });
-  console.log('WE GOT A RESULT', result);
-  return result;
+async function updateAlbumStatus(active) {
+  try {
+    await db
+      .collection('albums')
+      .doc(albumId)
+      .set({
+        active,
+      },
+      { merge: true });
+      console.log('success!');
+  } catch (err) {
+    console.error(err);
+    throw new Error('Could not update album status.');
+  }
 }
 
-async function createAlbum(userToken, userId, title) {
+async function activateAlbum(userToken, albumId) {
```

Comment:
Is this the function that "submits" the album for review by the tagged users?

Reply:
yeep